pkg/controller/jenkins/configuration/base/resources/route.go

```diff
@@ -0,0 +1,57 @@
+package resources
+
+import (
+	"github.com/jenkinsci/kubernetes-operator/pkg/apis/jenkins/v1alpha2"
+	routev1 "github.com/openshift/api/route/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+//RouteKind the kind name for route
+const RouteKind = "Route"
+//RouteAPIIsDiscoverable true when the Route API is installed and discoverable
+var RouteAPIIsDiscoverable = false
+
+func init ()  {
+	RouteAPIIsDiscoverable,_ = IsRouteAPIDiscoverable()
```

Comment:
err is not handled
please call `IsRouteAPIDiscoverable` in main.go and set RouteAPIIsDiscoverable in this package

Reply:
I changed teh visibility of var IsRouteAPIDiscoverable which is better. And now, the a call to IsRouteAPIDiscoverable will initialise it properly.